companies/giga.json

```diff
@@ -0,0 +1,20 @@
+{
+    "slug": "giga",
+    "relevant-countries": [
+        "de"
+    ],
+    "name": "GIGA.de",
+    "address": "℅ Ströer Media Brands GmbH\nTorstraße 49\n10119 Berlin-Mitte",
```

Comment:
I think the correct formatting should be:
`℅ Ströer Media Brands GmbH\nTorstraße 49\n10119 Berlin\nDeutschland`

Reply:
Yep, exactly. +1